package.json

```diff
@@ -37,9 +37,9 @@
     "build": "concurrently \"yarn buildBabel\" \"yarn buildTsc\"",
     "build:watch": "concurrently \"yarn buildBabel:esm -- --watch\" \"yarn buildTsc -- --watch\"",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "storybook dev -p 6006",
     "start": "concurrently \"yarn build:watch\" \"yarn storybook -- --no-manager-cache --quiet\"",
-    "build-storybook": "build-storybook",
+    "build-storybook": "storybook build",
```

Comment:
assuming it's failing because this build command was updated. lemme poke around netlify for a sec

Reply:
updated to `yarn build && yarn build storybook` in netlify but local `yarn build` is failing

Reply:
That update was made by their automatic upgrade script.. hmm

Reply:
okay, the build errors are resolved AFAICT @adamfratino

Reply:
It looks like "useAddonState" from client-api is deprecated, but I don't see it supported anywhere else, so I've re-added that package. https://github.com/storybookjs/storybook/issues/8518